### PR TITLE
Add metrics upstream

### PR DESCRIPTION
**Pull request by malnick**

### CODE REVIEW

**Comment by malnick** [comment]
@spahl This is ready for review.

**Comment by vespian** [comment]
Look good!

Would it be a problem to also add `proxy_set_header X-Forwarded-Proto $scheme;` to the location stanzas just as a good measure ? It will most probably not be used by the upstream, but it would be nice if we started thinking about a common set of headers we always pass to upstreams.

Thanks!

**Comment by malnick** [comment]
Can do!

**Comment by malnick** [comment]
@vespian This is updated.

File: nginx.agent.conf [hunk 1]
```diff
@@ -31,5 +35,12 @@ http {
             proxy_pass http://pkgpanda/;
             proxy_redirect http://$http_host/ /pkgpanda/;
         }
+
+        location /system/metrics {
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+						proxy_set_header X-Forwarded-Proto $scheme;
```
Comment (vespian) [inline nginx.agent.conf]: Could you please change the tabs into spaces here and down below ?

**Comment by malnick** [comment]
@vespian This should be ready now.

**Pull request merged**